r/tests/testthat/test-dplyr-mutate.R

```diff
@@ -140,17 +140,6 @@ test_that("transmute() with unsupported arguments", {
   )
 })
 
-test_that("transmute() defuses dots arguments (ARROW-13262)", {
```

Comment:
We should keep this test, right? Just change the function to something made up maybe.

Reply:
I don't think we need it. We solved the underlying issue (transmute not defusing the ellipsis arg). It we want the test in, we need a function for which the fallback to R works, so it can't be a made up one. We do not have a corresponding test for `mutate()`, for example.

Reply:
I used a call to `stringr::str_squish()` (which I don't think we will have bindings for anytime soon), so this is, technically, resolved.